include/alpaka/core/CudaHipMath.hpp

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2019-2021 Benjamin Worpitz, René Widera, Sergei Bastrakov
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <alpaka/core/BoostPredef.hpp>
+#include <alpaka/core/UniformCudaHip.hpp>
+
```

Comment:
Note: in the [original duplicated piece](https://github.com/alpaka-group/alpaka/blob/6861f7b31ef0725cb117d523b94fbe2c53b60b81/include/alpaka/math/abs/AbsUniformCudaHipBuiltIn.hpp#L16-L38) there was also checks and errors for backend and compiler mismatch. However, exactly the same checks are done in `alpaka/core/UniformCudaHip.hpp`, so I did not put them in this file as well.

Reply:
Fine for me!